tests/test_stack.py

```diff
@@ -80,5 +80,5 @@ def test_stack_tree(hlwm):
         - Client <windowid> "bash"
       - Frame Layer
         - Window <windowid>
-'''
+'''  # noqa: W291 ignore trailing whitespace in this string
```

Comment:
The suppression would have been reasonable, but I took the opportunity to fix the underlying reason for the trailing space. As soon as #396 is merged, you can drop this noqa.

Reply:
Removed it